onedal/basic_statistics/basic_statistics.cpp

```diff
@@ -162,8 +207,12 @@ ONEDAL_PY_INIT_MODULE(basic_statistics) {
     ONEDAL_PY_INSTANTIATE(init_compute_ops, sub, policy_spmd, task_list);
 #else // ONEDAL_DATA_PARALLEL_SPMD
     ONEDAL_PY_INSTANTIATE(init_compute_ops, sub, policy_list, task_list);
-    ONEDAL_PY_INSTANTIATE(init_compute_result, sub, task_list);
+    ONEDAL_PY_INSTANTIATE(init_partial_compute_ops, sub, policy_list, task_list);
+    ONEDAL_PY_INSTANTIATE(init_finalize_compute_ops, sub, policy_list, task_list);
 #endif // ONEDAL_DATA_PARALLEL_SPMD
+
+    ONEDAL_PY_INSTANTIATE(init_compute_result, sub, task_list);
```

Comment:
move all lines inside conditoinal

Reply:
are you sure that it is correct?  in my understanding compute_result should be initialized anyway

Reply:
im not 100% sure it will fix CI but we do not have any ONEDAL_PY_INSTANTIATE outside of these conditionals anymore since https://github.com/intel/scikit-learn-intelex/pull/1702

Reply:
i'm sure that ci failure doesn't related to this place. nevertheless, i would say that it is mistake in https://github.com/intel/scikit-learn-intelex/pull/1702 which should be fixed. do we have any tests for spmd? in my understanding current main branch shouldn't work if ONEDAL_DATA_PARALLEL_SPMD defined